desk.py

```diff
@@ -0,0 +1,88 @@
+import threading
+
+'''
+This class will essentially be a model for a desk, containing desk name, id, etc.
+Note: pin information will be stored in desk_map
+'''
+
+
+class Desk:
+    __id = -1
+    __desk_name = ""
+    __occupied = False
+    __desk_observers = None
```

Comment:
Since these fields are initialized in the constructor, I don't think this is necessary.